napari/layers/_image_layer/model.py

```diff
@@ -351,35 +317,42 @@ def interpolation_functions(self):
     def _clim_range_default(self):
         return [float(self.image.min()), float(self.image.max())]
 
-    def get_value(self, position, indices):
+    def get_value(self):
         """Returns coordinates, values, and a string for a given mouse position
         and set of indices.
 
-        Parameters
+        Returns
         ----------
-        position : sequence of two int
-            Position of mouse cursor in canvas.
-        indices : sequence of int or slice
-            Indices that make up the slice.
+        coord : tuple of int
+            Position of cursor in image space.
+        value : int or float or sequence of int or float
+            Value of the data at the coord.
+        """
+        coord = np.round(self.coordinates).astype(int)
+        coord[-2:] = np.clip(coord[-2:], 0,
+                             np.asarray(self._image_view.shape) - 1)
 
-        Returns
+        value = self._image_view[tuple(coord[-2:])]
+
+        return coord, value
+
+    def get_message(self, coord, value):
+        """Generates a string based on the coordinates and information about
```

Comment:
Docstring summaries/parameter descriptions should not redundantly list what type it returns/takes if possible.

```suggestion
        """Generate a status message based on the coordinates and information about
```

Reply:
fixed